examples/tutorials/Introduction_to_Bioinformatics.ipynb

```diff
@@ -294,8 +294,11 @@
         "id": "hy1zvhrAxycG"
```

Comment:
<p>Can you add a reference here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>